public/app/partials/testplan/testplan.html

```diff
@@ -1,13 +1,14 @@
 <h3 id="statistic_header">"{{ name }}" statistics</h3>
 <div ng-if="charts">
     <highchart config="charts[0]" ng-if="chartPercentType === 'duration'"></highchart>
-    <highchart config="charts[1]" ng-if="chartPercentType === 'failed'"></highchart>
+    <highchart config="charts[2]" ng-if="chartPercentType === 'failed'"></highchart>
+    <highchart config="charts[1]" ng-if="chartPercentType === 'metrics'"></highchart>
     <div ng-if="chartPercentType === 'number'">
         <div class="col-md-6">
-            <highchart config="charts[1]"></highchart>
+            <highchart config="charts[2]"></highchart>
```

Comment:
Maybe we need to start using object instead of array for charts? It will make code cleaner.